src/v3/src/util/theme.ts

```diff
@@ -11,115 +11,296 @@
+      set(theme, 'palette.error.dark', customTokens.PaletteDangerDark ?? dark);
+      mergedTokens.PaletteDangerLighter = lighter;
+      mergedTokens.PaletteDangerLight = light;
+      mergedTokens.PaletteDangerMain = main;
+      mergedTokens.PaletteDangerDark = dark;
+    }
+    if (customTokens.PaletteWarningMain) {
+      const {
+        lighter, light, main, dark,
+      } = generatePalette(customTokens.PaletteWarningMain);
+      set(theme, 'palette.warning.lighter', customTokens.PaletteWarningLighter ?? lighter);
+      set(theme, 'palette.warning.light', customTokens.PaletteWarningLight ?? light);
+      set(theme, 'palette.warning.main', customTokens.PaletteWarningMain ?? main);
+      set(theme, 'palette.warning.dark', customTokens.PaletteWarningDark ?? dark);
+      mergedTokens.PaletteWarningLighter = lighter;
+      mergedTokens.PaletteWarningLight = light;
+      mergedTokens.PaletteWarningMain = main;
+      mergedTokens.PaletteWarningDark = dark;
+    }
+    if (customTokens.PaletteSuccessMain) {
+      const {
+        lighter, light, main, dark,
+      } = generatePalette(customTokens.PaletteSuccessMain);
+      set(theme, 'palette.success.lighter', customTokens.PaletteSuccessLighter ?? lighter);
+      set(theme, 'palette.success.light', customTokens.PaletteSuccessLight ?? light);
+      set(theme, 'palette.success.main', customTokens.PaletteSuccessMain ?? main);
+      set(theme, 'palette.success.dark', customTokens.PaletteSuccessDark ?? dark);
+      mergedTokens.PaletteSuccessLighter = lighter;
+      mergedTokens.PaletteSuccessLight = light;
+      mergedTokens.PaletteSuccessMain = main;
+      mergedTokens.PaletteSuccessDark = dark;
+    }
+    set(theme, 'mixins.borderRadius', mergedTokens.BorderRadiusMain);
+    set(theme, 'mixins.borderStyle', mergedTokens.BorderStyleMain);
+    set(theme, 'mixins.borderWidth', mergedTokens.BorderWidthMain);
+    set(theme, 'palette.text.disabled', mergedTokens.TypographyColorDisabled);
+    set(theme, 'palette.text.primary', mergedTokens.TypographyColorBody);
+    set(theme, 'shadows[1]', mergedTokens.ShadowScale0);
+    set(theme, 'shadows[2]', mergedTokens.ShadowScale1);
+    set(theme, 'typography.body1.fontFamily', mergedTokens.TypographyFamilyBody);
+    set(theme, 'typography.body1.fontSize', mergedTokens.TypographySizeBody);
+    set(theme, 'typography.body1.lineHeight', mergedTokens.TypographyLineHeightBody);
+    set(theme, 'typography.body2.fontFamily', mergedTokens.TypographyFamilyBody);
+    set(theme, 'typography.body2.fontSize', mergedTokens.TypographySizeBody);
+    set(theme, 'typography.body2.lineHeight', mergedTokens.TypographyLineHeightBody);
+    set(theme, 'typography.button.fontFamily', mergedTokens.TypographyFamilyButton);
+    set(theme, 'typography.caption.color', mergedTokens.TypographyColorSubordinate);
+    set(theme, 'typography.caption.fontFamily', mergedTokens.TypographySizeSubordinate);
```

Comment:
Is this supposed to be `TypographySizeSubordinate`? Wouldn't this end up setting a font size value for the `font-family` property?

Reply:
Good catch. Definitely an oversight